mult.py

```diff
@@ -0,0 +1,32 @@
+def least_common_multiple(big_num, small_num):
+    count = 1
+    while True:
+        if (big_num * count)%small_num == 0:
+            return big_num * count
+        else:
+            count+=1
+
+
+# read numbers from user
+while True:
+    try:
+        a = int(input('enter first number\n'))
+    except ValueError:
+        print('enter valid input')
+        continue
+    else:

```

Comment:
Try and avoid `else` on `try-except` blocks. It's usually harder to understand.
Here for example, you can just call `break` after the `a = int...` line.

Reply:
I removed the user input part but understood the comment. Done